WordPress/gutenberg
# Add `home` to list of new templates

## What?

Adds 'Home' (home.html) to the list of templates that can be added in the Site editor.

<img width="250" alt="image" src="https://user-images.githubusercontent.com/1645628/214384178-5db638c9-e084-4b06-b663-8bd3be3d1999.png">

## Why?

Closes https://github.com/WordPress/gutenberg/issues/46325.

The Home template is often used in themes, and without it being included in the default list of templates, it needs to be created manually outside of the Editor.

However, I'm not sure if this highlights the confusion around the Front Page and Home templates too much. This is an existing problem, but perhaps listing the templates so close to each other is even more confusing. Still, I think it would be good to allow this template to be added without needing to leave the Editor. It would be great to know what everyone thinks.

## How?

Adds `home` to `DEFAULT_TEMPLATE_SLUGS`, and uses the `postList` icon for the Home icon.

## Testing Instructions




1. Go to the Site editor
2. Click the + to add a new template
3. 'Home' should display in the templates list, under Front Page

### Testing Instructions for Keyboard


## Screenshots or screencast
